tests/testthat/_snaps/build-articles.md

```diff
@@ -1,14 +1,116 @@
-# warns about missing images
+# links to man/figures are automatically relocated
+
+    Code
+      copy_figures(pkg)
+
+---
+
+    Code
+      build_articles(pkg, lazy = FALSE)
+
+# warns about missing images [plain]
 
     Code
       build_articles(pkg)
-    Output
-      -- Building articles -----------------------------------------------------------
-      Writing 'articles/index.html'
-      Reading 'vignettes/html-vignette.Rmd'
-      Writing 'articles/html-vignette.html'
     Condition
       Warning:
       Missing images in 'vignettes/html-vignette.Rmd': 'foo.png'
       i pkgdown can only use images in 'man/figures' and 'vignettes'
 
+# warns about missing images [ansi]
```

Comment:
In general I don't think it's worth testing these variations since you're effectively testing cli code, increasing coupling between the two packages.

Reply:
By default `cli::test_that_cli()` tests all 4 variations.

I can set e.g. `configs = c("plain")` every time I call `test_that_cli()`, but I don't think there's a way to set a global option for the `configs` value.

Reply:
I didn't even know that function existed 😄 I think you can just use regular snapshot tests without that function.

Reply:
IME if you use regular `test_that()`, then none of the cli outputs are captured in snapshots.

https://cli.r-lib.org/reference/test_that_cli.html

I have used `cli::test_that_cli()` in ~10 places to capture the outputs.

Reply:
That's not some intrinsic property of testthat/cli:

``` r
library(testthat)

test_that("success", {
  local_edition(3)
  
  expect_snapshot({
    cli::cli_alert_success("wow")
  })
})
#> Can't compare snapshot to reference when testing interactively.
#> ℹ Run `devtools::test()` or `testthat::test_file()` to see changes.
#> Current value:
#> Code
#>   cli::cli_alert_success("wow")
#> Message
#>   ✔ wow
#> ── Skip: success ───────────────────────────────────────────────────────────────
#> Reason: empty test
```

<sup>Created on 2023-11-27 with [reprex v2.0.2.9000](https://reprex.tidyverse.org)</sup>

Do you want me to look into why the output isn't getting captured in pkgdown?